### PR TITLE
add docsify to the projects list.

### PR DESCRIPTION
**Pull request by nisarhassan12**
This is how it looks:  
image

Fixes gitpod-io/contribute.dev#64

### CODE REVIEW

File: src/data/github-projects.json [hunk 1]
```diff
@@ -177,6 +177,21 @@
             "title": "Postwoman",
             "description": "A free, fast & beautiful API request builder (web alternative to Postman).",
             "language": "Javascript",
+            "tags": [
+                {
+                    "href": "https://gitter.im/docsifyjs/Lobby?utm_source=share-link&utm_medium=link&utm_campaign=share-link",
```
Comment (jankeromnes) [inline src/data/github-projects.json]: I think you're adding the DocsifyJS gitter chat to the Postwoman project.

File: src/data/github-projects.json [hunk 2]
```diff
@@ -177,6 +177,21 @@
...
+            "repoURL": "https://github.com/liyasthomas/postwoman",
+            "logo": "post-woman.png"
+        },
+        {
+            "title": "Docsify",
+            "description": "A magical documentation site generator.",
+            "language": "Javascript",
             "tags": [
                 {
                     "href": "https://github.com/liyasthomas/postwoman/blob/master/CONTRIBUTING.md",
```
Comment (jankeromnes) [inline src/data/github-projects.json]: Here I think you're adding Postwoman's`CONTRIBUTING.md`to the Docsify project.

**Comment by nisarhassan12** [comment]
> Thanks a lot for adding Docsify!
> 
> However, there seems to have been a mixup between Postwoman and Docsify. Please double-check the tags.

Thanks! I have made the changes.

**Comment by nisarhassan12** [comment]
> LGTM, many thanks! tada
> 
> But before we merge this, maybe let's wait for your setup improvements in [docsifyjs/docsify#1073](https://github.com/docsifyjs/docsify/pull/1073) to be merged? I think it's important that new open source contributors get a fully smooth & frictionless onboarding experience on all https://contribute.dev projects

Thanks! I agree :slightly_smiling_face:

**Comment by jankeromnes** [comment]
That’s awesome! 👍 Go for launch 🚀

**Pull request merged**